packages/flutter_test/lib/src/test_compat.dart

```diff
@@ -232,8 +232,8 @@ void test(
 /// avoid this flag if possible, and instead use the test runner flag `-n` to
 /// filter tests by name.
 @isTestGroup
-void group(Object description, Function body) {
-  _declarer.group(description.toString(), body);
+void group(Object description, Function body, dynamic skip) {
```

Comment:
should 'skip' be optional named here, too?

Reply:
D'oh